locksmith/src/routes/metadata.ts

```diff
@@ -1,8 +1,8 @@
 import express from 'express'
 import signatureValidationMiddleware from '../middlewares/signatureValidationMiddleware'
+import MetaDataController from '../controllers/metadataController'
```

Comment:
```suggestion
import MetadataController from '../controllers/metadataController'
```

Not critical, just a naming suggestion to have it match the namespace definition and be more like the file name.

Reply:
sounds good, i'll make this change and adjust its usage below